pkgs/tools/X11/imwheel/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, libX11, libXext, libXi, libXmu, libXt, libXtst }:
+
+stdenv.mkDerivation rec {
+  name = "imwheel-1.0.0pre12";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/imwheel/${name}.tar.gz";
+    sha256 = "2320ed019c95ca4d922968e1e1cbf0c075a914e865e3965d2bd694ca3d57cfe3";
+  };
+
+  buildInputs = [ libX11 libXext libXi libXmu libXt libXtst ];
+
+  postPatch = ''
+    substituteInPlace Makefile.in --replace "ETCDIR = " "ETCDIR = $out"
```

Comment:
Will this program need the rc file at `/etc/X11/imwheel` at runtime?

Reply:
I thought it was only a sample, but based on the docs it's probably sourced first:
```
       $HOME/.imwheelrc                                                                                                                    
            The users configuration file.                                                                                                  
                                                                                                                                           
       /etc/X11/imwheel/imwheelrc                                                                                                          
            The global location for the configuration                                                                                      
            file, it is always loaded.  Overided by                                                                                        
            the users configuration file.
```

Reply:
Running it in debug mode `-D`, it doesn't mention loading the system wide file, so it may not use it.  If I move my person file out of the way, and run with debug:
```
getRC:filename="/home/james/.imwheelrc"
uid=1001 euid=1001
gid=100 egid=100
/home/james/.imwheelrc: No such file or directory
/home/james/.imwheelrc: file permissions for your REAL user do not allow you to read it!
getRC:filename="/etc/X11/imwheel/imwheelrc"
uid=1001 euid=1001
gid=100 egid=100
/etc/X11/imwheel/imwheelrc: No such file or directory
/etc/X11/imwheel/imwheelrc: file permissions for your REAL user do not allow you to read it!
```

so I think it's fallback only, but we should fix the path.

Reply:
You'd probably need to look at the source for this but, for this to work simply, it has to look for this configuration file within the nix store. This only installs it there.

Then someone can just override it with their user config as usual.

Reply:
Now replacing /etc path in util.c